### PR TITLE
docs(instances): add unified snapshot MTA-2064

### PR DESCRIPTION
**Pull request by bene2k1**
### Description
Add unified snapshots to doc

### CODE REVIEW

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 1]
```diff
@@ -39,6 +45,7 @@ The snapshot features creates backups of specific volumes only. If you wish to c
 4. Complete the following steps in the wizard: 
     - Choose a **name** for the snapshot, or use the default suggested name.
     - Select the **volume** you wish to snapshot. All of your Instances' volumes will be shown on the drop-down list, grouped by Availability Zone.
+    - Select the snapshot type. Snapshots can either be of LSSD or Unified type.
```
Comment (ofranc) [inline compute/instances/how-to/create-a-snapshot.mdx:48]:
```suggestion
    - Select the snapshot type. Snapshots can either be of LSSD or unified type.
```

**Review by ofranc — APPROVED** [review]

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 2]
```diff
@@ -28,6 +28,12 @@ The snapshot features creates backups of specific volumes only. If you wish to c
 
 ## How to create a snapshot
 
+A snapshot takes a picture of a specific volume at one point in time. Creating snapshots of your volumes gives you total freedom of which volumes you want to back up, while images are more convenient for full backups of your Instance.  
+A unified snapshot can be used to create either a BSSD or LSSD volume.
+
+* **LSSD:** Choose this snapshot type to be able to create another local volume from your snapshot. Learn more
```
[the remark this answers is not in the document]
Comment (bene2k1) [reply compute/instances/how-to/create-a-snapshot.mdx:34]:
```suggestion
* **LSSD:** Choose this snapshot type to be able to create another local volume from your snapshot.
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by nedaehm — APPROVED** [review]

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 3]
```diff
@@ -12,6 +12,12 @@ dates:
...
 
+Several options are available to create snapshots from your Instance: 
+
+* **LSSD (Local storage):** This snapshot type can be created from local volumes. LSSD snapshots can be converted to local volumes only.
+* **BSSD (Block storage):** This snapshot type can be created from block volumes. BSSD snapshots can be converted to block volumes only.
+* **Unified:** This snapshot type can be created from either local or block volumes. Unified snapshots can be converted into either LSSD or BSSD volumes. 
+
 <Message type="note">
 
 The snapshot features creates backups of specific volumes only. If you wish to create a backup image which is a complete backup of your Instance including **all** of its volumes, consider using the [backup feature](/compute/instances/how-to/create-a-backup)
```
Comment (yrapaport-scw) [inline compute/instances/how-to/create-a-snapshot.mdx:23]:
```suggestion
The snapshot feature creates backups of specific volumes only. If you wish to create a backup image that is a complete backup of your Instance including **all** of its volumes, consider using the [backup feature](/compute/instances/how-to/create-a-backup).
```

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 4]
```diff
@@ -39,6 +47,7 @@ The snapshot features creates backups of specific volumes only. If you wish to c
 4. Complete the following steps in the wizard: 
     - Choose a **name** for the snapshot, or use the default suggested name.
     - Select the **volume** you wish to snapshot. All of your Instances' volumes will be shown on the drop-down list, grouped by Availability Zone.
+    - Select the snapshot type. The available types of snapshots are depending on your volume type and can either be LSSD, BSSD, or Unified.
```
Comment (yrapaport-scw) [inline compute/instances/how-to/create-a-snapshot.mdx:50]:
```suggestion
    - Select the snapshot type. The available types of snapshots depend on your volume type and can either be LSSD, BSSD, or Unified. 
```

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 5]
```diff
@@ -28,6 +34,8 @@ The snapshot features creates backups of specific volumes only. If you wish to c
 
 ## How to create a snapshot
 
+A snapshot takes a picture of a specific volume at one point in time. Taking snapshots of your volumes gives you total freedom of which volumes you want to back up. On the other hand, images are more convenient for full backups of your Instance.
```
Comment (yrapaport-scw) [inline compute/instances/how-to/create-a-snapshot.mdx:37]: Redundant with the Note above?

File: compute/instances/how-to/create-a-snapshot.mdx [hunk 6]
```diff
@@ -57,9 +66,15 @@ You can create a [volume](/compute/instances/concepts/#volume) from a snapshot a
 
   <Lightbox src="scaleway-volume-from-snapshot.webp" alt="" />
 
-5. Enter a name for the volume (or use the default suggested name), and click **Create Volume from Snapshot**
+5. Enter a name for the volume (or use the default suggested name), select the volume type, and click **Create Volume from Snapshot**.
+
+  <Message type="note">
+
+  The volume types available may change according to the type of snapshot you select.
```
Comment (yrapaport-scw) [inline compute/instances/how-to/create-a-snapshot.mdx:73]:
```suggestion
  The available volume types may change according to the type of snapshot you select. 
```

**Pull request merged**